homeassistant/components/fritzbox/sensor.py

```diff
@@ -94,6 +97,62 @@ class FritzSensorEntityDescription(
         suitable=lambda device: device.has_powermeter,  # type: ignore[no-any-return]
         native_value=lambda device: device.energy / 1000 if device.energy else 0.0,
     ),
+    # Thermostat Sensors
+    FritzSensorEntityDescription(
+        key="comfort_temperature",
+        name="Comfort Temperature",
+        native_unit_of_measurement=TEMP_CELSIUS,
+        device_class=DEVICE_CLASS_TEMPERATURE,
+        suitable=lambda device: device.has_thermostat
+        and device.comfort_temperature is not None,
+        native_value=lambda device: device.comfort_temperature,  # type: ignore[no-any-return]
+    ),
+    FritzSensorEntityDescription(
+        key="eco_temperature",
+        name="Eco Temperature",
+        native_unit_of_measurement=TEMP_CELSIUS,
+        device_class=DEVICE_CLASS_TEMPERATURE,
+        suitable=lambda device: device.has_thermostat
+        and device.eco_temperature is not None,
+        native_value=lambda device: device.eco_temperature,  # type: ignore[no-any-return]
+    ),
+    FritzSensorEntityDescription(
+        key="nextchange_temperature",
+        name="Next Scheduled Temperature",
+        native_unit_of_measurement=TEMP_CELSIUS,
+        device_class=DEVICE_CLASS_TEMPERATURE,
+        suitable=lambda device: device.has_thermostat
+        and device.nextchange_temperature is not None,
+        native_value=lambda device: device.nextchange_temperature,  # type: ignore[no-any-return]
+    ),
+    FritzSensorEntityDescription(
+        key="nextchange_time",
+        name="Next Scheduled Change Time",
+        device_class=DEVICE_CLASS_TIMESTAMP,
+        suitable=lambda device: device.has_thermostat
+        and device.nextchange_endperiod is not None,
+        native_value=lambda device: datetime.fromtimestamp(
+            device.nextchange_endperiod
+        ).isoformat(),
```

Comment:
use the `utc_from_timestamp()` from `homeassistant.util.dt`, those users timezone will be respected on UI